tests/unit/forklift/test_legacy.py

```diff
@@ -56,6 +58,28 @@
 )
 
 
+def _get_tar_testdata(compression_type=""):
+    temp_f = io.BytesIO()
+    with tarfile.open(fileobj=temp_f, mode="w:" + compression_type) as tar:
```

Comment:
Style nit: I saw you've used an f-string in a similar code. Perhaps we could change `"w:" + compression_type` to `f"w:{compression_type}"`?